public-ui/src/assets/db-images/All_Of_Us_Logo.svg

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is duplicating all (or most?) of our assets. Do you actually need all/any of these? If so, we should probably move this to a common location.

Reply:
/assets/db-images are what Lou gave me for databrowser. We could have some shared assets in common-ui at some point. But until design is finalized , i prefer to leave it as is.  

The assets/images folder i will delete though because I do not use those anymore.